### ISSUE TITLE
Create RRFS_NA retro capability

### ISSUE THREAD

**Issue by JeffHamiltonNOAA** [NOAA-GSL/VxLegacyIngest#429]
Haidao and Trevor have asked for the ability to do retro verification of RRFS_NA. Specifically they want:

UA
surface
ceiling
precip
reflectivity

**Comment by JeffHamiltonNOAA** [comment]
UA, ceiling, and surface are working on Hera. Reflectivity and Precip are coming next, then all will be replicated on Jet

**Comment by JeffHamiltonNOAA** [comment]
UA, ceiling, and surface are working on Jet. Precip and reflectivity are taking more time

**Comment by JeffHamiltonNOAA** [comment]
I have reflectivity up and running for RRFS_NA_13km on Hera. Working on precip, then will move over to Jet. RRFS_NA_3km will follow

**Comment by JeffHamiltonNOAA** [comment]
All capabilities are ready on Jet and Hera, I'm just waiting on some test data from Trevor to ensure that the 3km directory convention is properly set up.

**Comment by JeffHamiltonNOAA** [comment]
This issue is no longer relevant with updated RRFS domains. Closing
